library/Zend/Validate/File/Extension.php

```diff
@@ -134,7 +132,6 @@ public function getExtension()
      */
     public function setExtension($extension)
     {
-        $this->_extension = null;
```

Comment:
I think
```
$this->_extension = '';
```
is more appropriate

Reply:
Hello @fascinosum 
The property `$_extension` is `''` by default

Reply:
Only right after initialization of the object

Reply:
done